_data/team/roles/non_core_library_maintenance.yml

```
@@ -13,52 +13,53 @@ subroles:
 
   - subrole: distopia
     current_members:
-      - Richard Gowers
-      - Hugo MacDermott-Opeskin
-      - Rocco Meli
+      - "[Richard Gowers](https://github.com/richardjgowers)"
+      - "[Hugo MacDermott-Opeskin](https://github.com/hmacdope)"
+      - "[Rocco Meli](https://github.com/RMeli)"
 
   - subrole: cookiecutter-mdakit
     current_members:
-      - Lily Wang
-      - Irfan Alibay
+      - "[Lily Wang](https://github.com/lilyminium)"
+      - "[Irfan Alibay](https://github.com/IAlibay)"
 
   - subrole: mda-encore
-    current_members: []
+    current_members:
```

The MDA-ENCORE kit also has dedicated external maintainers, see https://github.com/MDAnalysis/mdaencore/blob/0ebcaff79d2f7309712d2fe2c1d21cfa55f47963/pyproject.toml#L24-L27 and I think we should list them here in addition to the MDA liaison
```suggestion
    current_members:
      - "[Kristoffer Enøe Johansson](https://github.com/enoee)" (external developer)
      - "[Kresten Lindorff-Larsen](https://github.com/lindorff-larsen)" (external developer)
```

---

@yuxuanzhuang you out yourself down here as "in charge" so please add the suggestion if you think that it's appropriate to list the external maintainers